release-doc-in-process/all_vf.TTL

```diff
@@ -260,6 +264,30 @@ vf:under  a          owl:ObjectProperty ;
         rdfs:range   vf:Agreement ;
         rdfs:comment        "Reference an agreement between agents which specifies the rules or policies which govern this event." .
 
+vf:numeratorFlow a   owl:ObjectProperty ;
+        rdfs:label   "vf:numeratorFlow" ;
+        rdfs:domain  vf:ExchangeRate ;
+        rdfs:range   [ owl:unionOf (vf:EconomicEvent vf:Commitment vf:Intent ) ] ;
+        rdfs:comment "References flow related to the numerator value" .
+
+vf:denominatorFlow a   owl:ObjectProperty ;
+        rdfs:label   "vf:denominatorFlow" ;
+        rdfs:domain  vf:ExchangeRate ;
+        rdfs:range   [ owl:unionOf (vf:EconomicEvent vf:Commitment vf:Intent ) ] ;
+        rdfs:comment "References flow related to the denominator value" .
+
+vf:numerator a   owl:DataProperty ;
+        rdfs:label   "vf:numerator" ;
+        rdfs:domain  xsd:decimal ;
+        rdfs:range   [ owl:unionOf (vf:EconomicEvent vf:Commitment vf:Intent ) ] ;
+        rdfs:comment "Numerator value" .
+
+vf:denominator a   owl:DataProperty ;
+        rdfs:label   "vf:denominator" ;
+        rdfs:domain  xsd:decimal ;
+        rdfs:range   [ owl:unionOf (vf:EconomicEvent vf:Commitment vf:Intent ) ] ;
+        rdfs:comment "Denominator value" .
+
```

Comment:
Not sure I understand the model.  Are the domain and range what you intend?

Reply:
Good catch! I've just fixed it...